### PR TITLE
Dr 7918 searchagent component style

### PR DESCRIPTION
**Pull request by maxstrebel**
* Adds search-agent variant
* Opposing to briefing I (for now) only created one variant

### CODE REVIEW

File: docs/components/search-agent.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Default Search Agent in horizontal alignment
+
+{% example html %}
+<div class="card">
+  <div class="card-block search-agent">
+    <div class="row">
+      <div class="col-12 col-sm-6 push-sm-6">
+        <div class="search-agent-body">
+          <p class="search-agent-subtitle">Suchagent</p>
```
Comment (jayhof) [inline docs/components/search-agent.md]: I'm not to happy about the naming here, "... subtitle" being mentioned before the "title". Maybe "subtitle" could be "tag" or "byline"

File: docs/components/search-agent.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          <h4 class="search-agent-title">Direkt in Deinen Posteingang</h4>
```
Comment (jayhof) [inline docs/components/search-agent.md]: We probably should not use a header tag here.

File: docs/components/search-agent.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          <p class="search-agent-text">Sobald ein neuer Job eingestellt wird, schicken wir Dir eine Mail.</p>
+          <a href="#" class="btn btn-primary">Benachrichtigung anlegen</a>
```
Comment (jayhof) [inline docs/components/search-agent.md]: Could we change the a tag to a button?

**Review by jayhof — CHANGES_REQUESTED** [review]

File: docs/components/search-agent.md (hunk 3 above)
Comment (maxstrebel) [reply docs/components/search-agent.md]: Agree on the other ones, at this one though, I think as it leads to another page the link might be the right element. Why would you prefer a button? Just so I know how to use it in the future.

File: docs/components/search-agent.md (hunk 1 above)
Comment (maxstrebel) [reply docs/components/search-agent.md]: You're right. I'd go for byline then.

File: docs/components/search-agent.md (hunk 3 above)
Comment (jayhof) [reply docs/components/search-agent.md]: The intent here is not to link to another page but rather trigger an action like showing a modal so the user only has to enter his e-mail address and be done and happy :)

**Review by jayhof — APPROVED** [review]

**Pull request merged**